### DISCUSSION TITLE
Snowflake Connector Restriction

### DISCUSSION THREAD

**Discussion by glick1** [hasura/graphql-engine#9315] [Question] [answered]
I see the Snowflake connector is only available for EE customers. Seems counterproductive to win potential Snowflake customers.

**Comment by dameleney** [comment, marked as answer]
The Hasura Snowflake connector is available to try on any Cloud edition for now. We are in the process of making it easy to try the connector in Hasura Enterprise trials.

We believe in the power and availability of open-source software and also want to be able to keep contributing so much value to the Hasura open-source community for free.  To be able to continue and build on our open-source history though, we need to build software that isn't open-source, enables us to pay our employees, and have a good ROI for our investors, so we can continue to improve all of the Hasura GraphQL Engine.
